ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -281,7 +287,7 @@ class ActionBar extends React.PureComponent {
             return;
           }
 
-          return TaskclusterModel.submit({
```

Comment:
Also, particularly, in this case, I had to remove ```return``` for linting to pass because of arrow functions not allowing return statement inside after re-enabling```consistent-return```.

Is this the correct way to go about this?

Reply:
@camd what do you think?

Reply:
Yeah, this is fine.  I tried without that return and it has no effect.  Not sure why it was there, tbh.  :)  So what you have done looks good here.